user-management/server/lib/authorization-middleware.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { NextFunction, Request, Response } from 'express';
+import jwt from 'jsonwebtoken';
+import { ClientError } from './client-error.js';
+
+const hashKey = process.env.TOKEN_SECRET ?? '';
+if (!hashKey) throw new Error('TOKEN_SECRET not found in env');
+
+export function authMiddleware(
+  req: Request,
+  res: Response,
+  next: NextFunction
+): void {
+  /* your code here */
+  const auth = req.get('Authorization');
```

Comment:
I had you change this from `req.headers.Authorization` to `req.get()`. While `req.get()` is the preferred technique for getting request headers, Tev used `req.headers.authorization`, which works as well. The capitalization is what made your code not work.